### PR TITLE
Allow upload file via URL

### PR DESCRIPTION
**Pull request by quanpython**

### CODE REVIEW

File: studio/app/common/routers/files.py [hunk 1]
```diff
@@ -112,3 +118,65 @@ async def create_file(workspace_id: str, filename: str, file: UploadFile = File(
...
+)
+async def get_download_status(workspace_id: str, file_name: str):
+    filepath = join_filepath([DIRPATH.INPUT_DIR, workspace_id, file_name])
+    try:
+        return DOWNLOAD_STATUS[filepath]
+    except:  # noqa
+        raise HTTPException(status_code=404)
+
+
+@router.get(
```
Comment (itutu-tienday) [inline studio/app/common/routers/files.py:139]: @quanpython 
Wouldn't POST be appropriate here?

Comment (quanpython) [reply studio/app/common/routers/files.py:139]: @itutu-tienday
I think POST would be more appropriate and secure. What do you think?

Comment (itutu-tienday) [reply studio/app/common/routers/files.py:139]: @quanpython 
In terms of the meaning of the request (registering data into the system), I think POST is appropriate.
If there is little need to use GET, please use POST.

**Review by ReiHashimoto — APPROVED** [review]
LGTM

**Pull request merged**